src/main/frontend/ui.cljs

```diff
@@ -368,17 +370,18 @@
 
 (defn scroll-down?
   []
-  (let [scroll-top (get-scroll-top)]
-    (let [down? (> scroll-top @last-scroll-top)]
-      (reset! last-scroll-top scroll-top)
-      down?)))
+  (let [scroll-top (get-scroll-top)
+        down? (>= scroll-top @last-scroll-top)]
+    (reset! last-scroll-top scroll-top)
+    down?))
 
 (defn on-scroll
-  [node on-load on-top-reached]
+  [node {:keys [on-load on-top-reached threhold]
```

Comment:
Did we mean threshold? I don't think this effects functionality but happy to fix typo if we want

Reply:
Yeah, I mean threshold, just pushed a fix.